### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by filipacarreira**
Testing module

### CODE REVIEW

File: life_expectancy/tests/test_cleaning.py [hunk 1]
```diff
@@ -1,16 +1,45 @@
 """Tests for the cleaning module"""
```
Comment (joaogomes98) [inline life_expectancy/tests/test_cleaning.py:1]: This is a suggestion. It is good practice to separate your tests by modules. So, my suggestion would be to create separate testing modules for each module (data cleaning, data loading/saving and main).

**Review by joaogomes98 — APPROVED** [review]
Apart from the minor suggestion I made, this testing integration was spot on! Everything is well written, the code is well structured, and, for someone who wasn't used to conduct tests, it is working as intended. Good job!

File: life_expectancy/tests/test_cleaning.py (hunk 1 above)
Comment (filipacarreira) [reply life_expectancy/tests/test_cleaning.py:1]: Thank you for your suggestion, already changed it!! Can we merge?

File: life_expectancy/tests/test_main.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"""Tests for the main module"""
+import pandas as pd
+
+from life_expectancy.main import main
+
+def test_main(pt_life_expectancy_expected):
```
Comment (MrCordeiro) [inline life_expectancy/tests/test_main.py:6]:
```suggestion
@patch("life_expectancy.main.load_data")
@patch("life_expectancy.main.save_data")
def test_main(mock_save_data,  mock_load_data, pt_life_expectancy_expected):
```

**Review by MrCordeiro — CHANGES_REQUESTED** [review]

File: life_expectancy/tests/test_main.py (hunk 2 above)
Comment (MrCordeiro) [reply life_expectancy/tests/test_main.py:6]: mock_load_data.return_value = MY_INPUT_FIXTURE

Comment (MrCordeiro) [reply life_expectancy/tests/test_main.py:6]: mock_save_data.side_effect = print("Being saved!")

**Pull request merged**